bedrock/firefox/templates/firefox/features/index.html

```diff
@@ -129,8 +129,8 @@ <h3 class="mzp-c-card-picto-title">{{ ftl('features-shared-better-bookmarks') }}
       </section>
     </div>
 
-    {% if ftl_has_messages('features-index-fingerprinter-blocking', 'features-index-fingerprinting-is-a', 'features-index-translate-the-web', 'features-index-translate-more-than') %}
```

Comment:
Just confirming the logic here: 

We now will show each feature individually, if localized, rather than blocking the entire section from showing if any of the features were not localized. 

Is that correct?

Reply:
Correct. The downside is that there's a possibility of some locales having none of the three and the page would render with an empty `div` but I can live with that. The other option is to wrap the entire section in a complex condition so it adds the `div` if any one of the three is translated, but that just seemed like overkill.